src/plugin/breakpoints.ts

```diff
@@ -1,15 +1,33 @@
-import { LOCAL_VARIABLES_PREFIXES, PAGE_IDS } from './constants';
+import { LOCAL_VARIABLES_PREFIXES, PAGE_IDS, THEME_PREFIXES } from './constants';
 import { Page } from './page';
 
 export class Breakpoints extends Page {
-  private modeId: string;
+  private modeId?: string;
 
   constructor(modeId: string) {
     super(PAGE_IDS.BREAKPOINTS);
     this.modeId = modeId;
   }
```

Comment:
nit: this can be simplified to:

```suggestion
  constructor(private modeId?: string) {
    super(PAGE_IDS.BREAKPOINTS);
  }
```